src/renderer/dx12.rs

```diff
@@ -1071,12 +1083,12 @@ impl RenderEngine {
                 display_pos[1] + display_size[1],
             ];
 
-            [[2. / (r - l), 0., 0., 0.], [0., 2. / (t - b), 0., 0.], [0., 0., 0.5, 0.], [
-                (r + l) / (l - r),
-                (t + b) / (b - t),
-                0.5,
-                1.0,
-            ]]
+            [
+                [2. / (r - l), 0., 0., 0.],
+                [0., 2. / (t - b), 0., 0.],
+                [0., 0., 0.5, 0.],
+                [(r + l) / (l - r), (t + b) / (b - t), 0.5, 1.0],
+            ]
```

Comment:
This should be formatted in the original way via `cargo +nightly fmt --all`.

Reply:
I have it explicitly in my editor configuration to use the `+nightly` flag to `cargo fmt` but it comes and goes for me as well for some reason -- I caught this specific one because lately my `rust_analyzer` has been applying the same change automatically over and over with me only noticing the diff in commits.